src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -550,6 +684,113 @@ struct ContentView: View {
 
 </InlineFilter>
 
+<InlineFilter filters={['flutter']}>
+
+Start by adding a new state to track the todos and the ability to fetch the todo list when it first renders:
```

Comment:
This feels like a web way of expressing the situation. I think we should just call it a list.
```suggestion
Start by adding a new list to track the todos and the ability to fetch the todo list when it first renders:
```